server/middlewares/UuidValidator.js

```diff
@@ -0,0 +1,25 @@
+import checkUUID from './validations/validationHelper';
```

Comment:
@ekpangmichael please rename this file to `uuidValidator.js`

Reply:
Okay